services-turf/src/main/java/com/mapbox/turf/TurfConversion.java

```diff
@@ -163,4 +176,83 @@ public static double convertLength(@FloatRange(from = 0) double distance,
     }
     return radiansToLength(lengthToRadians(distance, originalUnit), finalUnit);
   }
+
+  /**
+   * Takes a {@link FeatureCollection} and
+   * returns all positions as {@link Point} objects.
+   *
+   * @param featureCollection a {@link FeatureCollection} object
+   * @return a new {@link FeatureCollection} object with {@link Point} objects
+   * @since 4.7.0
+   */
+  public static FeatureCollection explode(@NonNull FeatureCollection featureCollection) {
+    List<Feature> featuresForFinalFeatureCollection = new ArrayList<>();
+    for (Feature singleFeature : featureCollection.features()) {
+      for (int i = 0; i < explode(singleFeature.geometry()).features().size(); i++) {
+        featuresForFinalFeatureCollection.add(explode(singleFeature.geometry()).features().get(i));
+      }
+    }
+    return FeatureCollection.fromFeatures(featuresForFinalFeatureCollection);
+  }
+
+  /**
+   * Takes a {@link Feature}  and
+   * returns its position as a {@link Point} objects.
+   *
+   * @param feature a {@link Feature} object
+   * @return a new {@link FeatureCollection} object with {@link Point} objects
+   * @since 4.7.0
+   */
+  public static FeatureCollection explode(@NonNull Feature feature) {
+    return explode(feature.geometry());
+  }
+
+  /**
+   * Takes a {@link Geometry} object, checks what type of geometry it is,
+   * and then returns all positions as {@link Point} objects.
+   *
+   * @param geometry a {@link Geometry} object
+   * @return a new {@link FeatureCollection} object with {@link Point} objects
+   * @since 4.7.0
+   */
+  private static FeatureCollection explode(@NonNull Geometry geometry) {
+    List<Feature> featuresForFinalFeatureCollection = new ArrayList<>();
```

Comment:
Looks like there is a method: 
https://turfjs.org/docs/#coordAll  
that is supposed to work for FeatureCollection/Feature and Geometry.
We ported only the Geometry parameter. 
What do you think if we port coordAll(FeatureCollection) and coordAll(Feature) and then take advantage of it here.

Reply:
>What do you think if we port coordAll(FeatureCollection) and coordAll(Feature) and then take advantage of it here.

Yea, I think we should do this. I've needed it a bunch during my work on porting more methods to Turf for Java.

Reply:
>Yea, I think we should do this. 

https://github.com/mapbox/mapbox-java/pull/1009

Reply:
@osana , just refactored and rebased this pr. Added usage of the new `coordAll()` methods, which were added in https://github.com/mapbox/mapbox-java/pull/1009.